package.json

```diff
@@ -15,17 +15,12 @@
     "vuex": "^4.0.0"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^18.0.0",
-    "@rollup/plugin-node-resolve": "^11.2.1",
-    "@rollup/plugin-typescript": "^8.2.1",
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
     "@vitejs/plugin-vue": "^1.2.1",
     "@vue/compiler-sfc": "^3.0.11",
     "@vue/eslint-config-typescript": "^7.0.0",
     "eslint": "^7.24.0",
-    "eslint-plugin-vue": "^7.9.0",
```

Comment:
this one is needed i think?

Reply:
![image](https://user-images.githubusercontent.com/24670327/116534212-71dde000-a90c-11eb-906c-0cf4be11c502.png)